prime-router/src/main/kotlin/fhirengine/translation/hl7/FhirToHl7Converter.kt

```diff
@@ -151,7 +151,7 @@ class FhirToHl7Converter(
         focusResources.forEachIndexed { index, singleFocusResource ->
             // The element context must now get the focus resource
             elementContext.focusResource = singleFocusResource
-            if (canEvaluate(element, bundle, singleFocusResource, elementContext)) {
+            if (canEvaluate(element, bundle, singleFocusResource, focusResource, elementContext)) {
```

Comment:
What do you think of renaming `singleFocusResource` (and maybe `focusResources` too) for clearer distinction between that and `focusResource`? Maybe `elementResource`/`elementResources`?

Reply:
Yep, can do